buildkite/src/Command/MinaArtifact.dhall

```diff
@@ -184,23 +184,6 @@ let docker_step
                   ]
                 , TestExecutive = [] : List DockerImage.ReleaseSpec.Type
                 , LogProc = [] : List DockerImage.ReleaseSpec.Type
-                , Toolchain =
-                  [ DockerImage.ReleaseSpec::{
-                    , service = Artifacts.Type.Toolchain
-                    , network = Network.lowerName Network.Type.Devnet
-                    , deb_codename = spec.debVersion
-                    , deb_profile = spec.profile
-                    , build_flags = spec.buildFlags
-                    , deb_repo = DebianRepo.Type.Local
-                    }
-                  ]
-                , ItnOrchestrator =
```

Comment:
Itn orchestrator also is not built currently form mina artifact job but it got its own